yb-voyager/cmd/importData.go

```diff
@@ -390,6 +390,9 @@ func updateTargetConfInMigrationStatus() {
 }
 
 func importData(importFileTasks []*ImportFileTask) {
+	if importerRole == TARGET_DB_IMPORTER_ROLE {
```

Comment:
I would say, put this in `importDataCommandFn` ? We set other things in MSR such as `updateFallForwardEnabledInMetaDB` .